### ISSUE TITLE
dataset 

### ISSUE THREAD

**Issue by Tianxzzz** [RManLuo/GSNOP#1]
The MOOC dataset in the given google cloud disk does not contain the file：edge_feature.pt

which cause b.edata['f'], and b.srcdata['h'] is error

**Comment by RManLuo** [comment]
Hi @1874tian, As we discussed in the paper, no edge features are provided for the MOOC dataset. Please add `--rand_edge_features 128` to randomly generate edge features when running the script.

**Comment by Tianxzzz** [comment]
Thanks for your answer. I have solved the problem.

By the way, I have another question , is the earlystop of all experiments set to 5? 
 When I was running  python train_np.py --data  WIKI_0.1 --config config/TGAT.yml --base_model snp --ode --eval_neg_samples 50, the result is :Old AP: 0.32830166006699196  Overall test ap:0.307492 Overall test mrr:0.487012. But in Table 3, the result should be AP:0.3366 MRR:0.5240.
In addtion, which AP is shown in paper, the Old AP or the Overall test ap?

**Comment by RManLuo** [comment]
Yes, the early stop is set to 5 for all experiments and overall AP is reported in the paper. But, after discussing with some colleges, I found the old AP used in [tgl](https://github.com/amazon-science/tgl/blob/9447fbd0887f4cff5dcc5a5d4fff4055aa964fe5/train.py#L117) could be a more proper way to evaluate the performance in dynamic graphs.

For the results, our framework is based on the neural process, in which we sample a function at each prediction. This could lead to some stochastic factors in the results. Besides, we implemented our code based on [tgl](https://github.com/amazon-science/tgl), which randomly sampled negative edges for evaluation in a C++ module. We cannot freeze the random seed in it. This could also introduce some variances to the results.
